src/main/java/seedu/lifeasier/commands/DeleteNotesCommand.java

```diff
@@ -49,16 +50,16 @@ private void findTitle(Ui ui, NoteList notes, String title, NoteHistory noteHist
             ui.showMultipleMatchesFoundMessage();
 
             logger.log(Level.INFO, "Start of printing all matching notes");
-            ui.printMultipleNoteMatches(notes, title);
+            matchArr = ui.printMultipleNoteMatches(notes, title, matchArr);
```

Comment:
Maybe you can try revamp this to use an ArrayList instead of Array. If it's too much then it's ok